openstudiocore/src/model/AvailabilityManagerNightCycle.cpp

```diff
@@ -486,20 +539,20 @@ AvailabilityManagerNightCycle::AvailabilityManagerNightCycle(const Model& model)
 
   // Cooling Control Zone List
   ModelObjectList coolingControlThermalZoneList = ModelObjectList(model);
-  controlThermalZoneList.setName(this->name().get() + " Cooling Control Zone List");
+  coolingControlThermalZoneList.setName(this->name().get() + " Cooling Control Zone List");
   ok = setPointer(OS_AvailabilityManager_NightCycleFields::CoolingControlZoneorZoneListName, coolingControlThermalZoneList.handle());
   OS_ASSERT(ok);
 
   // Cooling Control Zone List
```

Comment:
comment is wrong.

Reply:
@kbenne please do fix that stuff when you catch it. I can't push to develop, so I'd need to a open a new PR to fix it after the fact